tests/unit/test_base_command.py

```diff
@@ -13,10 +14,15 @@
 from pip._internal.utils.temp_dir import TempDirectory
 from tests.lib.path import Path
 
+FIXED_TIME_VALUE = 1547704837.040001
+FIXED_TIME_FORMAT = datetime.fromtimestamp(FIXED_TIME_VALUE).strftime(
+    "%Y-%m-%dT%H:%M:%S,%f"
+)[:-3]
+
 
 @pytest.fixture
 def fixed_time(utc: None) -> Iterator[None]:
```

Comment:
Perhaps the `utc` fixture should be removed, since it doesn't work on Windows (and shouldn't be necessary anymore)?